src/bundle/Controller/Content/ContentTreeController.php

```diff
@@ -226,7 +226,7 @@ private function getLocationPermissionRestrictions(Location $location): array
                 'restrictedLanguageCodes' => $createLimitationsValues[Limitation::LANGUAGE],
             ],
             'edit' => [
-                'hasAccess' => $lookupUpdateLimitationsResult->hasAccess(),
```

Comment:
Have you checked other uses of this method? If it does not work correctly, it should be fixed, or all usage should be replaced.